common-api/src/main/java/org/pmiops/workbench/cdr/dao/DomainInfoDao.java

```diff
@@ -36,11 +36,11 @@
       // move these searches to a more suitable technology, e.g. Elasticsearch.
       "    UNION DISTINCT " +
       "    (select domain_id, concept_id from concept " +
-      "     where concept_id in (?2) and can_select=1)) c1 " +
+      "     where concept_id in (?3) or concept_code = ?2 and can_select=1)) c1 " +
```

Comment:
sorry @calbach. I totally forgot about this when you asked me the other day.
So, the idea was to show the concept count when searched with concept code irrespective of they being standard. 
The other search check needs the standard concept check.
So added it back in here.

Reply:
Ah, because of the `and standard_concept IN ('S', 'C')` above